### PR TITLE
Location breakdown

### PR DESCRIPTION
**Pull request by Buhle79**
skip validation for source USSD

### CODE REVIEW

File: tbconnect/tests/test_views.py [hunk 1]
```diff
@@ -102,6 +102,33 @@ def test_location_validation(self):
...
+        user = get_user_model().objects.create_user("test")
+        user.user_permissions.add(Permission.objects.get(codename="add_tbcheck"))
+        self.client.force_authenticate(user)
+        response = self.client.post(
+            self.url,
+            {
+                "msisdn": "27856454612",
+                "source": "USSD",
+                "province": "ZA-WC",
+                "city": "Cape Town",
```
Comment (bearnard) [inline tbconnect/tests/test_views.py:118]: is city optional?

Comment (Buhle79) [reply tbconnect/tests/test_views.py:118]: No it's not optional, it is mandatory.

Comment (Buhle79) [reply tbconnect/tests/test_views.py:118]: image

Comment (bearnard) [reply tbconnect/tests/test_views.py:118]: when we skip location, do we also skip city question?

Comment (Buhle79) [reply tbconnect/tests/test_views.py:118]: Yes

Comment (bearnard) [reply tbconnect/tests/test_views.py:118]: then the tbcheck POST will fail

**Comment by Buhle79** [comment]
Hi @bearnard for re review please

**Comment by bearnard** [comment]
👍

**Pull request merged**